### PR TITLE
style

### PR DESCRIPTION
**Pull request by sokrahta**
+ disable code coverage build failure
+ enforce style on compile
+ address most style issues in src/main
+ address some style issues in src/test (ignoring magic number and string literal for now)

### CODE REVIEW

File: query-coordinator-http/src/test/scala/com/socrata/querycoordinator/TestQueryRewriterDateTruncBase.scala [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
-    * - a soql statement.  Note this must be the mapped statement, ie. non-system columns prefixed by an _, and backtick escaped
+    * - a soql statement.  Note this must be the mapped statement,
+    *   i.e. non-system columns prefixed by an _, and backtick escaped
     * - a Seq of the soql types for each column in the rollup selection
     */
   val rollups = Seq(
-    ("r_ymd", "SELECT date_trunc_ymd(`_crim-date`), `:wido-ward`, count(*) GROUP BY date_trunc_ymd(`_crim-date`), `:wido-ward`"),
-    ("r_ym", "SELECT date_trunc_ym(`_crim-date`), `:wido-ward`, count(*) GROUP BY date_trunc_ym(`_crim-date`), `:wido-ward`"),
-    ("r_y", "SELECT date_trunc_y(`_crim-date`), `:wido-ward`, count(*) GROUP BY date_trunc_y(`_crim-date`), `:wido-ward`")
+    ("r_ymd", "SELECT date_trunc_ymd(`_crim-date`), `:wido-ward`, count(*) " +
```
Comment (znep) [inline query-coordinator-http/src/test/scala/com/socrata/querycoordinator/TestQueryRewriterDateTruncBase.scala]: Breaking this all up just to satisfy some line length constraint makes these tests a lot harder to write and work with and compare with each other.  Every change requires messing around with formatting and multiple lines... is there really benefit from breaking these up to meet some arbitrary rule?

Comment (sokrahta) [inline query-coordinator-http/src/test/scala/com/socrata/querycoordinator/TestQueryRewriterDateTruncBase.scala]: not feeling too strongly about test code, will revert those lines.

Comment (kaidamasaki) [inline query-coordinator-http/src/test/scala/com/socrata/querycoordinator/TestQueryRewriterDateTruncBase.scala]: Want to suppress the warning, too?

**Comment by sokrahta** [comment]
cool changes @chitang LGTM :shipit:

**Pull request merged**